src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
Fixed. Though this sounds a bit awful, why has `PREFIXOP` a precedence but then uses `below_DOT_AND_SHARP`? Can we fix that?